src/lib/serviceRoutinesV2/patchEntity.cpp

```diff
@@ -71,6 +71,12 @@ std::string patchEntity
   eP->id = compV[2];
   eP->type = ciP->uriParam["type"];
 
+  if (forbiddenIdChars(ciP->apiVersion, eP->id.c_str() , NULL))
+  {
+    OrionError oe(SccBadRequest, "invalid character in URI");
+    return oe.render(ciP, "");
+  }
+
```

Comment:
I'll check  other 'paths' with ID for this point. If any is found, we can create an issue for all them and solve them in the same PR  ...